### PR TITLE
Remove redundant semicolons in Python code

### PR DESCRIPTION
**Pull request by practicalswift**

### CODE REVIEW

**Comment by fanquake** [comment]
trivial ACK 5cdf106

File: contrib/devtools/clang-format-diff.py [hunk 1]
```diff
@@ -128,7 +128,7 @@ def main():
         line_count = int(match.group(3))
       if line_count == 0:
         continue
-      end_line = start_line + line_count - 1;
+      end_line = start_line + line_count - 1
```
Comment (MarcoFalke) [inline contrib/devtools/clang-format-diff.py]: Just a note that this exists upstream. Not against removing it here, though.

**Comment by MarcoFalke** [comment]
utACK 5cdf10611fe5ea77d4632c24e69e68aa9b8119e0

**Comment by paveljanik** [comment]
utACK https://github.com/bitcoin/bitcoin/commit/5cdf10611fe5ea77d4632c24e69e68aa9b8119e0

**Comment by laanwj** [comment]
Please, do not open a ton of pull requests to do minor style stuff changes for the test scripts. If those are necessary at all I'd prefer to see them combined over time (there's no hurry for these).

**Comment by practicalswift** [comment]
@laanwj OK, understood! Thanks for clarifying :-)

**Review by jonasschnelli — APPROVED** [review]
utACK 5cdf10611fe5ea77d4632c24e69e68aa9b8119e0

**Pull request merged**